keyboards/kira80/keymaps/via/rules.mk

```diff
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+RAW_ENABLE = yes
```

Comment:
```suggestion
```

Reply:
^ Specifically, VIA enables RAW, by default. So it's not necessary to enable both.

Reply:
ah thanks didn't know that :)